### DISCUSSION TITLE
AI-Keep learning!

### DISCUSSION THREAD

**Discussion by Timmy110403540** [1101-GS3073-C/Python-Community-GS3073-C#134] [General]
NVIDIA的執行長日前說明NVIDIA的AI發展已進步至理解字彙和句子，以及上下文。他也提及AI可以從程式設計的過程中學習常識，增進既有的智慧，來修正各種的錯誤。執行長也希望以後能利用自己產業，去貢獻社會-及減少產業的浪費。
相信AI已從1960年代的舊有發展至全新世代。現在AI竟然還有學習的功能，變得越來越聰明，且有著出錯率大幅降低的偉大成就。人手一AI設備已近在咫尺!

文章來源:https://finance.technews.tw/2021/11/30/nvidia-ai-can-learn-common-sense-from-the-programming-process/

**Comment by bonny123865** [comment]
我覺得ai的能力真的從以前到現在進步好多，感覺已經有一個正常人的能力了，而且出錯率又低！

**Comment by pocper1** [comment]
如果AI繼續開發，AI可能會比人類還要聰明!?

**Comment by briankuo123** [comment]
AI的發展真的不可限量，但是其中延伸出的社會道德問題跟法律規範也想必會很多。

**Comment by wh20502** [comment]
感覺有點像在炒股票，但是我對技術還是非常樂觀，會不會哪天AI是由AI教，然後變得超乎人類想像了。

**Comment by 110409538** [comment]
AI竟然也可以學習，那未來可以自己幫自己修復bug了。

**Comment by yasklop** [comment]
感覺以後AI能學會的東西有好多，未來是不是有很多人要失業了:(

**Comment by Shiba010** [comment]
隨著AI越來越厲害，會不會人類很多工作機會都要被取代了....
